mkdocs_src/site/404.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
`mkdocs_src/site` dir shouldn't be included.

Reply:
I have removed file and folder and updated the description

Reply:
Resolve conversations you have fixed.